### PR TITLE
backport trivial 2021 12 21

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/optional.h [hunk 1]
```diff
@@ -5,12 +5,21 @@
 #ifndef BITCOIN_OPTIONAL_H
 #define BITCOIN_OPTIONAL_H
 
+#include <utility>
```
Comment (UdjinM6) [inline src/optional.h:8]: 15292 was just merged via #4635

File: src/init.cpp [hunk 2]
```diff
@@ -661,7 +661,7 @@ void SetupServerArgs()
         "each level includes the checks of the previous levels "
         "(0-4, default: %u)", DEFAULT_CHECKLEVEL), ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::DEBUG_TEST);
     gArgs.AddArg("-checkmempool=<n>", strprintf("Run checks every <n> transactions (default: %u, regtest: %u)", defaultChainParams->DefaultConsistencyChecks(), regtestChainParams->DefaultConsistencyChecks()), ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::DEBUG_TEST);
-    gArgs.AddArg("-checkpoints", strprintf("Disable expensive verification for known chain history (default: %u)", DEFAULT_CHECKPOINTS_ENABLED), ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::DEBUG_TEST);
+    gArgs.AddArg("-checkpoints", strprintf("Enable rejection of any forks from the known historical chain until block 295000 (default: %u)", DEFAULT_CHECKPOINTS_ENABLED), ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::DEBUG_TEST);
```
Comment (UdjinM6) [inline src/init.cpp:664]: 17286: s/295000/1450000/

File: test/functional/test_framework/test_framework.py [hunk 3]
```diff
@@ -647,6 +647,11 @@ def skip_if_no_wallet(self):
         if not self.is_wallet_compiled():
             raise SkipTest("wallet has not been compiled.")
 
+    def skip_if_no_wallet_tool(self):
+        """Skip the running test if bitcoin-wallet has not been compiled."""
+        if not self.is_wallet_tool_compiled():
+            raise SkipTest("bitcoin-wallet has not been compiled")
```
Comment (UdjinM6) [inline test/functional/test_framework/test_framework.py:658]: 17497: dashify

File: test/functional/test_framework/test_framework.py [hunk 4]
```diff
@@ -666,6 +671,10 @@ def is_wallet_compiled(self):
 
         return config["components"].getboolean("ENABLE_WALLET")
 
+    def is_wallet_tool_compiled(self):
+        """Checks whether bitcoin-wallet was compiled."""
```
Comment (UdjinM6) [inline test/functional/test_framework/test_framework.py:680]: 17497: dashify

File: src/Makefile.bench.include [hunk 5]
```diff
@@ -56,7 +56,6 @@ bench_bench_dash_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
 bench_bench_dash_LDADD = \
   $(LIBBITCOIN_SERVER) \
   $(LIBBITCOIN_WALLET) \
-  $(LIBBITCOIN_SERVER) \
```
Comment (UdjinM6) [inline src/Makefile.bench.include:59]: 18429: requires 18397

File: src/wallet/rpcwallet.cpp [hunk 6]
```diff
@@ -4105,7 +4105,7 @@ UniValue walletcreatefundedpsbt(const JSONRPCRequest& request)
                                 {
                                     {"txid", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The transaction id"},
                                     {"vout", RPCArg::Type::NUM, RPCArg::Optional::NO, "The output number"},
-                                    {"sequence", RPCArg::Type::NUM, RPCArg::Optional::NO, "The sequence number"},
+                                    {"sequence", RPCArg::Type::NUM, /* default */ "depends on the value of the 'locktime' and 'options.replaceable' arguments", "The sequence number"},
```
Comment (UdjinM6) [inline src/wallet/rpcwallet.cpp:4108]: 19018: no `options.replaceable`

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: src/optional.h (hunk 1 above)
Comment (PastaPastaPasta) [reply src/optional.h]: rebased

File: src/init.cpp (hunk 2 above)
Comment (PastaPastaPasta) [reply src/init.cpp:664]: done, squashed

File: test/functional/test_framework/test_framework.py (hunk 3 above)
Comment (PastaPastaPasta) [reply test/functional/test_framework/test_framework.py]: done, squashed

File: src/Makefile.bench.include (hunk 5 above)
Comment (PastaPastaPasta) [reply src/Makefile.bench.include]: dropped

File: src/wallet/rpcwallet.cpp (hunk 6 above)
Comment (PastaPastaPasta) [reply src/wallet/rpcwallet.cpp]: dropped

File: test/functional/test_framework/test_framework.py (hunk 4 above)
Comment (PastaPastaPasta) [reply test/functional/test_framework/test_framework.py:680]: done, squashed

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: test/functional/test_framework/test_framework.py (hunk 4 above)
Comment (PastaPastaPasta) [reply test/functional/test_framework/test_framework.py]: actually done :see_no_evil: 

I'm just making sure you're actually reviewing :P

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**